Procfile.dev

```diff
@@ -0,0 +1,2 @@
+web: unset PORT && env RUBY_DEBUG_OPEN=true bin/rails server
```

Comment:
why `unset PORT` and why `RUBY_DEBUG_OPEN=true` ? do we want to merge this?

Reply:
what about
`web: bin/rails server -p3000` ?

Reply:
Yes, I think that's better for the port, the other config is not necessary.